python/astro_metadata_translator/translators/hsc.py

```diff
@@ -82,7 +82,7 @@ def can_translate(cls, header, filename=None):
             return header["INSTRUME"] == "Hyper Suprime-Cam"
 
         for k in ("EXP-ID", "FRAMEID"):
-            if k in header:
+            if cls.is_keyword_defined(header, k):
```

Comment:
Why do you call `is_keyword_defined()` here and not `is_key_ok()` like everywhere else? (I realise this is an odd question having just asserted that there is no difference in functionality)